packages/entities/entities-shared/src/components/entity-empty-state/EntityEmptyState.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+  <div class="kong-ui-public-entity-empty-state">
+    <div
+      v-if="$slots.icon"
+      class="entity-empty-state-icon"
+    >
+      <slot name="icon" />
+    </div>
+    <div
+      v-else
+      class="entity-empty-state-image"
+    >
+      <slot name="image" />
+    </div>
```

Comment:
Let's combine these into a single image slot. We don't need to differentiate between whether we're slotting an image or an icon.

Reply:
We'll need to differentiate in order to apply classes to style to icons v/s image?

Reply:
What styles do you think will differ? Won't it just be centered with the same bottom margin?

Reply:
Consistent icon size for icons, v/s height & width for image

Reply:
I think it's overkill to try to style the size here. Let's handle the positioning, but let the host app determine the size.